generators/environment/templates/outrigger/outrigger.yml

```diff
@@ -19,53 +19,66 @@ scripts:
     alias: run
     description: Run one of the build container services such as 'cli' or 'drush'.
     run:
-      - docker-compose -f $PWD/build.yml run --rm
-
-  setup:
-    alias: setup
-    description: Run the end-to-end repository initialization and site install script.
-    run:
-      - rig project run:welcome
-      - rig project sync:start
-      - start.sh
+      - if [ "$OSTYPE" == "linux-gnu" ]; then COMPOSE_FILE=build.devcloud.yml; else COMPOSE_FILE=build.yml; fi
+      - docker-compose run --rm
 
   up:
     alias: up
     description: Start up operational docker containers and filesystem sync.
     run:
       - rig project sync:start
+      - if [ "$OSTYPE" == "linux-gnu" ]; then COMPOSE_FILE=docker-compose.yml; fi
```

Comment:
If it is not Linux, that implies it is probably macOS or Windows, in which case we want the default behavior of `-f docker-compose.yml -f docker-compose.override.yml`.